src/engraving/libmscore/harmony.cpp

```diff
@@ -2010,7 +2010,7 @@ void Harmony::addDegree(const HDegree& d)
 
 int Harmony::numberOfDegrees() const
 {
-    return _degreeList.size();
+    return static_cast<int>(_degreeList.size());
```

Comment:
I believe, better change method signature to `size_t Harmony::numberOfDegrees() const`

Reply:
This one is an easy change, will do.